tools/build_config/build.ps1

```diff
@@ -1,29 +1,73 @@
+<#
+.SYNOPSIS
+  This script is used to build, test and package Horace on Windows.
+
+.DESCRIPTION
+  This script requires Matlab, Visual Studio, CMake>=3.7 and CTest be installed
+  on your system and available on the path.
+
+  This script also requires that Herbert be findable by CMake. CMake will search
+  in common places for Herbert e.g. in the same directory as Horace.
+
+  Use "Get-Help ./build.ps1 -Detailed" for parameter descriptions.
+
+.EXAMPLE
+  ./build.ps1 -build
+  # Builds Horace
+.EXAMPLE
+  ./build.ps1 -test
+  # Runs all Horace tests
+.EXAMPLE
+  ./build.ps1 -package
+  # Packages Horace
+.EXAMPLE
+  ./build.ps1 -build -package -vs_version 2017 -matlab_release "R2019a"
```

Comment:
does `./build.ps1 -build -package -vs_version 2017 -matlab_release R2019a` (i.e. without the quotes) work?

Reply:
Yes!